crates/core/diesel.toml

```diff
@@ -6,4 +6,5 @@ file = "src/db/schema.rs"
 import_types = [
   "diesel::sql_types::*",
   "diesel_full_text_search::{TsVector as Tsvector, TsQuery as Tsquery}",
-]
+  "crate::db::custom_types::{SettingType as Settingtype , Mode}",
```

Comment:
```suggestion
  "crate::db::custom_types::{SettingType as Settingtype, Mode}",
```

Reply:
I don't know why GitHub has so much trouble with adding suggestions to this file, all that needs to happen here is removing the space before the comma.

Reply:
I always do this , it would be great if fmt formats that